### PR TITLE
[RFR]Test migration with edited mapping

### PR DESCRIPTION
**Pull request by sshveta**
{{ pytest: cfme/tests/v2v/test_v2v_migrations.py --use-provider rhv42 --use-provider vsphere67-nested --provider-limit 2 -vvvv --long-running -k test_migration_with_edited_mapping}}

Test migration with edited mapping test case .

### CODE REVIEW

**Comment by jawatts** [comment]
@sshveta 5.9 and 5.10 failing PRT:
```
cfme/fixtures/v2v.py:349: TypeError
TypeError
'function' object has no attribute '__getitem__'
```

**Comment by sshveta** [comment]
> @sshveta 5.9 and 5.10 failing PRT:
> 
> ```
> cfme/fixtures/v2v.py:349: TypeError
> TypeError
> 'function' object has no attribute '__getitem__'
> ```

Fixed that

**Comment by jawatts** [comment]
Waiting on PRT

File: cfme/tests/v2v/test_v2v_migrations.py [hunk 1]
```diff
@@ -729,3 +729,53 @@ def _cleanup():
...
     migrated_vm = get_migrated_vm_obj(vm_obj, v2v_providers.rhv_provider)
     assert vm_obj.mac_address == migrated_vm.mac_address
+
+
+@pytest.mark.ignore_stream("5.9")
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]], indirect=True)
+def test_migration_with_edited_mapping(request, appliance, v2v_providers, edited_form_data,
+                                       form_data_vm_obj_single_datastore,
+                                       host_creds, conversion_tags, soft_assert):
+    _form_data, edited_form_data = edited_form_data
```
Comment (jawatts) [inline cfme/tests/v2v/test_v2v_migrations.py]: please add new polarion metadata to this. 
```
Polarion:
        assignee: 
        caseimportance: 
        initialEstimate: 
```

**Comment by sshveta** [comment]
Test is passing locally . Timed out for request on PRT .
Retesting ..

**Pull request merged**